data/config-schema/src/main/resources/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json

```diff
@@ -22,7 +22,7 @@
               "description": "The maximum number of cached flow sandboxes.",
               "type": "integer",
               "minimum": 0,
-              "default": 10
+              "default": 20
```

Comment:
I think we need to be careful with this.
While I'm sure increasing this is a good idea (and maybe it should be much much higher (100+) in a production system, increasing this to get our tests to pass is masking a problem rather than fixing it.
The same issues will be hit at a higher load.

CC @simon-johnson-r3

Reply:
Yes, I agree. The underlying problem is that we do not handle flow execution very well when sandbox cache eviction happens. This PR was just an attempt to improve the current situation with E2E tests.
An alternative could be to make E2E tests run less in parallel by using `@Isolated` or `SAME_THREAD` annotations.